x-pack/plugins/security_solution/server/endpoint/routes/actions/response_actions.ts

```diff
@@ -298,7 +298,7 @@ function responseActionRequestHandler<T extends EndpointActionDataParameterTypes
     //        `getter` function), we need to include this additional validation here, since
     //        `agent_type` is included in the schema independent of the feature flag
     if (
-      req.body.agent_type &&
+      req.body.agent_type === 'sentinel_one' &&
```

Comment:
Good catch here.
I initially was thinking that if `agent_type` was in the API request and this feature (for this release) was disabled, then use of that field is not allowed and thus we should error. But since the default value in the schema for this is always `endpoint`, I know see that was not the right approach.  We probably should have a test for it.